src/components/Search/SearchView.tsx

```diff
@@ -22,8 +23,14 @@ function SearchView(props: SearchViewProps) {
     useFocusInput();
 
   const navigate = useNavigate();
-  const goToCompareResultsPage = () => {
-    navigate('/compare-results', { replace: false });
+
+  const goToCompareResultsPage = (selectedRevisions: Revision[]) => {
+    const revisions = [...selectedRevisions.map(rev => truncateHash(rev.revision))];
+    const repos = [...selectedRevisions.map(rev => rev.repository_id)];
```

Comment:
nit: you don't need to do the spread operation in `[...XXX.map()]` because `map` always returns a new array.

Reply:
If this is not needed I'd prefer that it be removed